src/style.css

```diff
@@ -1 +1,124 @@
 /* add styles here */
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  font-style: normal;
+  src: url('fonts/Avenir-Book.ttf') format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  font-style: normal;
+  src: url('fonts/Avenir-Heavy.ttf') format('truetype');
+}
+
+* {
+  margin: 0;
```

Comment:
It's not recommended to use the `*` selector to set styles for all elements, as it can negatively affect performance. Try to apply these styles specifically to the elements that need them.